Makefile

```diff
@@ -507,9 +507,11 @@ govet:
     ./test/k8sT/... \
     ./tools/...
 
-ineffassign:
-	@$(ECHO_CHECK) ineffassign
-	$(QUIET) ineffassign .
+lint:
+	#TODO remove once Vagrant version is updated
```

Comment:
Has this TODO been resolved?

Reply:
@christarazi  I have one related PR cilium/packer-ci-build#211. However, as packer PR removes ineffassign, so to avoid any issue, would love to address this TODO and bump vagrant version in another PR (once cilium/packer-ci-build#211 is merged)

Reply:
Once cilium/packer-ci-build#211 is merged we will also need to trigger a new build of the vagrant box and then bump the vagrant box versions in this repo. So IMO it is fine to leave this TODO for now and remove it as part of the future PR bumping the vagrant box version.

Reply:
I think this can be removed now, isn't it? I'm not exactly fond of a `make` target that `curl ... | sh ...`.

Reply:
Agree, the `TODO` comment and the `curl ... | sh` installation can now be removed from this PR. We bumped the vagrant box versions in #12261 and they now include `golangci-lint`.